src/core_plugins/timelion/public/app.js

```diff
@@ -86,12 +86,19 @@ app.controller('timelion', function (
     },
     run: function () {
       const title = savedSheet.title;
-      safeConfirm('Are you sure you want to delete the sheet ' + title + ' ?').then(function () {
+      function doDelete() {
         savedSheet.delete().then(() => {
           notify.info('Deleted ' + title);
           kbnUrl.change('/');
         }).catch(notify.fatal);
-      });},
+      }
+
+      const confirmModalOptions = {
+        onConfirm: doDelete,
+        confirmButtonText: 'Delete sheet'
+      };
+      confirmModal('Are you sure you want to delete the sheet ' + title + ' ?', confirmModalOptions);
```

Comment:
I know this isn't strictly part of this PR but since you are touching this line anyway, would you mind changing the string concatenation to a string template, per https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#use-template-strings-to-interpolate-variables-into-strings?

It would also make it consistent with some of the other interpolated confirmation strings you have used at other places in this PR.